pkg/server/helpers.go

```diff
@@ -349,3 +352,38 @@ func getUserFromImageUser(user string) (*int64, string) {
 	// If user is a numeric uid.
 	return &uid, ""
 }
+
+// getImageMetadata returns corresponding metadata of the image reference, if image is not
```

Comment:
We need to pull image when running pod sandbox.
We may need a better function name here though.